apps/storybook/src/Customization.stories.mdx

```diff
@@ -51,30 +51,31 @@ as you see fit. For instance, if you'd like to change the color of the curve of
+| ------------------------------ | -------------------------- | ------------------------------------------------------ |
+| `--h5w-canvas--bgColor`        | `transparent`              | Background color of canvas                             |
+| `--h5w-plotTitle--color`       | `inherit`                  | Text color of plot title                               |
+| `--h5w-plotTitle--fontFamily`  | `inherit`                  | Font family of plot title                              |
+| `--h5w-plotTitle--fontSize`    | `1.125em`                  | Font size of plot title                                |
+| `--h5w-plotTitle--fontWeight`  | `inherit`                  | Font weight of plot title                              |
+| `--h5w-ticks--color`           | `gray`                     | Color of ticks                                         |
+| `--h5w-tickLabels--color`      | `#333`                     | Text color of tick labels                              |
+| `--h5w-tickLabels--fontFamily` | `inherit`                  | Font family of tick labels                             |
+| `--h5w-tickLabels--fontSize`   | `inherit`                  | Font size of tick labels                               |
+| `--h5w-axisLabels--color`      | `#333`                     | Text color of axis labels                              |
+| `--h5w-axisLabels--fontFamily` | `inherit`                  | Font family of axis labels                             |
+| `--h5w-axisLabels--fontSize`   | `inherit`                  | Font size of axis labels                               |
+| `--h5w-axisLabels--fontWeight` | `600`                      | Font weight of axis labels                             |
+| `--h5w-grid--color`            | `gray`                     | Color of grid lines                                    |
+| `--h5w-grid--opacity`          | `0.33`                     | Opacity of grid lines                                  |
+| `--h5w-tooltip--color`         | `inherit`                  | Text color of tooltip                                  |
+| `--h5w-tooltip--bgColor`       | `rgba(255, 255, 255, 0.9)` | Background color of tooltip                            |
+| `--h5w-tooltip--fontFamily`    | `inherit`                  | Font family of tooltip                                 |
+| `--h5w-tooltip--fontSize`      | `0.75em`                   | Font size of tooltip                                   |
+| `--h5w-tooltip-guide--color`   | `gray`                     | Color of tooltip guide                                 |
+| `--h5w-tooltip-guide--width`   | `1.5`                      | Width of tooltip guide                                 |
+| `--h5w-tooltip-guide--opacity` | `0.5`                      | Opacity of tooltip guide                               |
+| `--h5w-error--color`           | `brown`                    | Color of error messages and UI elements in error state |
```

Comment:
Just moving `--h5w-error--color` to this section so I can rename the `UI` section below to `Toolbar`. This better matches the package's structure.